modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/workflow/analyzers/OrtResultAnalyzer.java

```diff
@@ -183,18 +183,20 @@ private Artifact mapArtifact(Optional<JsonNode> ortPackage, Optional<JsonNode> s
     }
 
     private Artifact mapArtifactFromAnalyzerResult(JsonNode ortPackage) {
-        Artifact a = new Artifact("OrtResult")
+        Artifact artifact = new Artifact("OrtResult")
                 .addFact(mapFileName(ortPackage))
                 .addFact(new DeclaredLicenseInformation(mapDeclaredLicenses("declared_licenses", ortPackage)))
                 .addFact(new ArtifactSourceUrl(mapSourceUrl(ortPackage)));
-        mapCoordinates(ortPackage.get("id").textValue()).ifPresent(a::addFact);
+        mapCoordinates(ortPackage.get("id").textValue()).ifPresent(artifact::addFact);
```

Comment:
is `"id"` always present or will this sometimes cause a NPE?

Reply:
id is always present